include/clangmetatool/include_graph_dependencies.h

```diff
@@ -0,0 +1,53 @@
+#ifndef INCLUDED_CLANGMETATOOL_INCLUDE_GRAPH_DEPENDENCIES_H
+#define INCLUDED_CLANGMETATOOL_INCLUDE_GRAPH_DEPENDENCIES_H
+
+#include <clangmetatool/collectors/include_graph_data.h>
+
+namespace clangmetatool {
+struct IncludeGraphDependencies {
```

Comment:
Can you add doc here? Just something simple about how these functions can be used to manipulate and check dependencies in an include graph

Reply:
Ack. Done.